src/main/resources/assets/paths/studyById.yaml

```diff
@@ -0,0 +1,23 @@
+get:
+  summary: Get Study by Id
+  description: Finds Study and Properties of the Study specified by id.
+  parameters:
+    - name: id
+      in: path
+      description: ID of the Study
+      required: true
+      schema:
+        type: integer
+  tags:
+    - Study
+  responses:
+    200:
+      description: The Study with properties
+      content:
+        application/json:
+          schema:
+            $ref: '../schemas/DatasetRegistrationSchemaV1.yaml'
```

Comment:
This should be`../schemas/Study.yaml` - it should exists, I think, but i'm not sure.

Reply:
It does exist and I added it, I wasn't sure which one was more appropriate based on the ticket description. We might need to make some edits in the future but for now this should work fine